CHANGELOG.adoc

```diff
@@ -44,7 +65,7 @@ To be transparent and set expectations right, we've applied a Beta label on any
 In the meantime, we will work hard to improve the Zettle Developer Platform. An example of the ongoing work is to improve the API documentation.
```

Comment:
Cannot figure out if this is displayed anywhere, but I would suggest the following wording:
"We are currently working to improve the Zettle Developer Platform, including the API documentation."

Reply:
Thanks @AndreaFilyo . I fixed your comment. :)